web_responsive/static/src/css/web_responsive.scss

```diff
@@ -131,16 +132,45 @@ $chatter_zone_width: 35%;
         }
     }
 }
+.o_main_navbar {
+    color: color-yiq($o-brand-odoo);
 
+    > ul > li > a,
+    > ul > li > label {
+        color: color-yiq($o-brand-odoo);
+    }
+}
 // Iconized full screen apps menu
 .o_menu_apps {
     .search-input:focus {
         border-color: $o-brand-primary;
     }
 
+    a.full {
+        width: $o-navbar-height;
+        text-align: center;
+    }
+
     .dropdown-menu.show {
+        visibility: visible;
+        opacity: 1;
+    }
+
+    .dropdown-menu {
         @include full-screen-dropdown();
 
+        visibility: hidden;
+        opacity: 0;
+        transition: visibility 50ms linear, opacity 50ms linear;
```

Comment:
You're using `ease` transition below, which is more elegant. Also, animating `visibility` is pointless because it's not a numeric value.

```suggestion
        opacity: 0;
        transition: opacity 50ms ease;
```

Reply:
When visibility is not animated strange behavior occurs in chrome.
![3163f00a-92e3-4bb5-a313-3702ff78b6ef](https://user-images.githubusercontent.com/17651502/89535888-70dcaf80-d7cd-11ea-88f5-4968471a90cc.gif)

But if I add the animation also to the visibility this no longer happens

![captured-min](https://user-images.githubusercontent.com/17651502/89536472-548d4280-d7ce-11ea-884b-83a58974196b.gif)

Reply:
I removed the transition to avoid sacrificing auto focus functionality, possibly in a future pull request I will add the transition keeping the auto focus.